app/views/references/_reference.html.erb

```diff
@@ -1,17 +1,24 @@
-<tr>
-  <td>
-    <%= reference.name %>
-  </td>
-  <td>
-    <span class="tel value"><%= reference.phone %></span>
-  </td>
-  <td>
-    <%= reference.email %>
-  </td>
-  <td>
-    <%= reference.relationship %>
-  </td>
-  <td>
-    <%= reference.whentocall %>
-  </td>
-</tr>
\ No newline at end of file
+<%= form_for @adopter do |f| %>
+  <%= f.fields_for :references, f.object.references.order(:name) do |rf| %>
+    <tr>
+      <td>
+        <%= rf.text_field :name %>
+      </td>
+      <td>
+        <span class="tel value"><%= rf.text_field :phone %></span>
+      </td>
+      <td>
+        <%= rf.text_field :email %></div>
+      </td>
+      <td>
+        <%= rf.text_field :relationship %>
+      </td>
+      <td>
+        <%= rf.text_field :whentocall %>
+      </td>
+      <td>
+        <%= rf.submit "Save", class: "btn btn-primary" %>
+      </td>
+    </tr>
+  <% end %>
+<% end %>
```

Comment:
From taking a look at how this appears on the site, it winds up looking really really wide.  Given all the small screens people are using  it might be worth considering a different layout, maybe something a little more vertical?